### ISSUE TITLE
v-model with select and text area issue

### ISSUE THREAD

**Issue by javialon26** [nuxt/nuxt.js#1817]
In SSR the select fields are not rendered with the selected attribute, then the selected option appears selected after all page load.
The same happens with the textarea field, first appears empty and then when the page loads, it shows the content of v-model.

**Comment by NicoPennec** [comment]
share github project or source code to reproduce, please...

**Comment by javialon26** [comment]
I did a test with a clean installation and the same thing happened.

Steps to reproduce:
1. install a nuxt project with vue-cli (vue init nuxt-community/starter-template <project-name>)
2. in pages/index.vue add:

```
<template>
  <form>
    <select v-model="selectedOption">
      <option value="" disabled></option>
      <option v-for="option in options" :value="option.id">{{ option.label }}</option>
    </select>
    <textarea v-model="text"></textarea>
  </form>
</template>
```

```
<script>
export default {
  data () {
    return {
      text: 'Hello world!',
      selectedOption: 'en',
      options: [ {id: 'es', label: 'Espanol'}, {id: 'en', label: 'Ingles'} ]
    }
  }
}
</script>
```
3. Run project and refresh browser.

In the select, first you see the option "Spanish" and then switch to English. In the textarea first it looks empty and then the text. This happens very quickly because it is a new project.

**Comment by Kasheftin** [comment]
`<select v-model="selectedOption">` equals with `<select :value="selectedOption" ..@change events..>`.
`<textarea v-model="text">` also converted to `<textarea :value="text" @input="...">`.

vue.js is smart. It understands that when you call `<select :value="something">`, you want to set the value of select tag to something. But HTML select tag itself does not have the value property. If you want to set the select value in plain html, you have to use `<select><option selected="selected" value="somevalue"></option></select>` construction. The same with textarea - the `value` property does not exist in plain HTML - you have to use `<textarea>somevalue</textarea>` instead.

That's about vue. But nuxt is not so smart. When nuxt performs SSR, it compiles `<select :value="something">` to the plain html like `<select value="valueofsomething">`. That's why we don't see textarea and select tags filled on page load. 

For now I would suggest to append the template with valid html properties like that:

    <select v-model="opt">
      <option v-for="option in options" :value="option.id" :selected="option.id===opt">{{option.label}}</option>
    </select>
    <textarea v-model="text">{{text}}</textarea>

This will render to the valid HTML with prefilled values.

**Comment by NicoPennec** [comment]
@javialon26  @Kasheftin note that it is not Nuxt that performs SSR, but [vue-server-renderer](https://github.com/vuejs/vue/tree/dev/packages/vue-server-renderer)!

**Comment by Atinux** [comment]
As @NicoPennec said, Nuxt.js uses official Vue.js modules to server-render your application :)

Actually, `vue-server-renderer` does not handle correctly the value into textarea, but you can easily mimic it like @Kasheftin showed you.

Example: https://glitch.com/edit/#!/future-kitty?path=pages/index.vue:8:9
Demo: https://future-kitty.glitch.me/

**Comment by javialon26** [comment]
@Atinux @NicoPennec thank you!

**Comment by javialon26** [comment]
fixed in vue v2.5.3 :)

**Comment by Atinux** [comment]
Good to know, great job by opening an issue on vue.js repo @javialon26
